### ISSUE TITLE
Unable to adjust population database AF

### ISSUE THREAD

**Issue by lynnpais** [broadinstitute/seqr#4301]
**Describe the bug**
I'm unable to set the population database AF using the dropdown. No matter what I select, it drops to zero. 

**Link to page(s) where bug is occurring**
https://seqr.broadinstitute.org/variant_search/family/F032092_rgp_2015

**Scope of the bug**
Checked an RGP case (Link above) as well as another non-RGP case

**Screenshots**
<img width="2000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK3hiQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--e45e1b73b3f890bf670fcf470652335a73198ded/Screenshot%202024-08-14%20at%205.57.06%E2%80%AFPM.png" alt="Screenshot 2024-08-14 at 5.57.06 PM.png" />

**Comment by hanars** [comment]
I am unable to replicate this, when I select something from the dropdown everything selects as expected. Can you please describe exactly what you are doing - did you select a saved search first? Did you edit anything else first?

**Comment by lynnpais** [comment]
When I submitted the ticket last eve, the population AF using the dropdown would only be zero no matter what value I selected from the dropdown. I didn't select any filters.

However this morning, when I first select any AF, it first goes to 0. When I try again, it works. Attaching a short clip to describe the issue. 

https://github.com/user-attachments/assets/c66d3316-0424-478f-8913-d4aebb16f818

**Comment by hanars** [comment]
This video is super helpful! I think this may be related to https://github.com/broadinstitute/seqr-private/issues/1489 so lets see if the fix for that fixes this

**Comment by hanars** [comment]
Okay this shoudl be fixed now, but since I was never able to replicate it I would need you to confirm

**Comment by lynnpais** [comment]
No, unfortunately this doesn't seem to be fixed. When I try to adjust the population frequency, it always sets to 0, regardless of what I choose. On the second try, it actually goes to the AF I select. 

Short clip to show you what's going on -

https://github.com/user-attachments/assets/e4ef4ba6-e6a1-412a-8d6e-186664d1a0e0
